### PR TITLE
Fix thumbstick press issue

### PR DESCRIPTION
**Pull request by ksuprynowicz**
This will fix https://github.com/overte-org/overte/issues/459

### CODE REVIEW

File: plugins/openvr/src/ViveControllerManager.cpp [hunk 1]
```diff
@@ -2068,6 +2068,12 @@ controller::Input::NamedVector ViveControllerManager::InputDevice::getAvailableI
 }
 
 QString ViveControllerManager::InputDevice::getDefaultMappingConfig() const {
-    static const QString MAPPING_JSON = PathUtils::resourcesPath() + "/controllers/vive.json";
+    QString name(getOpenVrDeviceName().c_str());
+    QString MAPPING_JSON;
+    if (name.contains(QString("Vive")) || name.contains(QString("Vive"))) {
```
Comment (JulianGro) [inline plugins/openvr/src/ViveControllerManager.cpp:2073]:
```suggestion
    if (name.contains(QString("Vive")) {
```
This seems redundant.

**Review by JulianGro — CHANGES_REQUESTED** [review]
Looks logical to me except for the one comment. GCC doesn't complain.

File: plugins/openvr/src/ViveControllerManager.cpp (hunk 1 above)
Comment (ksuprynowicz) [reply plugins/openvr/src/ViveControllerManager.cpp:2073]: Thank you! Second "Vive" was intended to be "HTC" and would cause a bug otherwise.

Comment (ksuprynowicz) [reply plugins/openvr/src/ViveControllerManager.cpp]: I fixed it in [dfef968](https://github.com/overte-org/overte/pull/481/commits/dfef9680f97cce6b65592fa6d5a97f251a17d53a)

**Pull request merged**